runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -298,7 +300,11 @@ public void onActorDeath(ActorDeath actorDeath)
 			Player player = (Player) actor;
 			if (player == client.getLocalPlayer() && config.screenshotPlayerDeath())
 			{
-				takeScreenshot("Death", SD_DEATHS);
+				final int regionID = WorldPoint.fromLocalInstance(client, client.getLocalPlayer().getLocalLocation()).getRegionID();
```

Comment:
Consider an early return here instead, more concise this way

```suggestion
final int regionID = WorldPoint.fromLocalInstance(client, client.getLocalPlayer().getLocalLocation()).getRegionID();
if (config.ignoreLMSDeaths() && LAST_MAN_STANDING_REGIONS.contains(regionID)) {
    return;
}

takeScreenshot("Death", SD_DEATHS);
```